lib/charms/prometheus_k8s/v0/prometheus_scrape.py

```diff
@@ -11,6 +11,13 @@
 shared between Prometheus charms and any other charm that intends to
 provide a scrape target for Prometheus.
 
+## Requirements
+
+Using this library requires you to fetch the juju_topology library from
+[observability-libs]((https://github.com/canonical/observability-libs/blob/main/lib/charms/observability_libs/v0/juju_topology.py).
```

Comment:
This doc is rendered on charmhub and by convention we try to have all links point to charmhub resources as well, when appropriate.

```suggestion
[observability-libs](https://charmhub.io/observability-libs/libraries/juju_topology).
```